pkg/test/resourcefixture/testdata/basic/eventarc/v1alpha1/eventarcgooglechannelconfig/eventarcgooglechannelconfig-minimal/_http.log

```diff
@@ -0,0 +1,67 @@
+GET https://eventarc.googleapis.com/v1/projects/${projectId}/locations/us-central1/googleChannelConfig?%24alt=json%3Benum-encoding%3Dint
+Content-Type: application/json
+User-Agent: kcc/${kccVersion} (+https://github.com/GoogleCloudPlatform/k8s-config-connector) kcc/controller-manager/${kccVersion}
+X-Goog-Request-Params: name=projects%2F${projectId}%2Flocations%2Fus-central1%2FgoogleChannelConfig
+
+200 OK
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
+
+{
+  "name": "projects/${projectId}/locations/us-central1/googleChannelConfig",
```

Comment:
Is this pre-created before the test?

Reply:
its a singleton. always exists in the cloud.,

Reply:
I see. Thanks!